lib/rules/no-array-prototype-extensions.js

```diff
@@ -145,6 +145,41 @@ function variableNameToWords(name) {
     .split(' ');
 }
 
+/**
+ * Returns the fixing object if it can be fixable otherwise returns an empty array.
+ *
+ * @param {Object} callExpressionNode The call expression AST node.
+ * @param {Object} fixer The ESLint fixer object which will be used to apply fixes.
+ * @returns {Object|[]}
+ */
+function applyFix(callExpressionNode, fixer, context) {
+  const propertyName = callExpressionNode.callee.property.name;
+
+  switch (propertyName) {
+    case 'any':
+      return fixer.replaceText(callExpressionNode.callee.property, 'some');
+    case 'filterBy': {
```

Comment:
how about put those method names into constants on top? `any`, `filterby` etc.

Reply:
Since those values are not used more than once, do you see value in making them constants?